docs/src/reference/workflow-creation.md

```diff
@@ -0,0 +1,89 @@
+# Workflow Instance Creation
+
+Depending on the workflow definition, an instance of it can be created in various ways.
+
+## Explicit creation of a workflow instance
+
+A workflow instance can be created by sending a command specifying the process id or the unique key of the workflow.
+There are two commands to create a workflow instance.
+
+### Asynchronous execution
+
+ A workflow that has a [none start event](/bpmn-workflows/none-events/none-events.html#none-start-events) can be started explicitly using the command [CreateWorkflowInstance](/reference/grpc.html#createworkflowinstance-rpc).
+ When the broker receives this commands, it creates a new workflow instance and immediately respond with the workflow instance id.
+ The execution of the workflow happens after the response is send.
+
+ <details>
+   <summary>Code example</summary>
```

Comment:
What do you think about using the Zbctl instead of the Java client? It is more language agnostic and could help non-Java users.